### PR TITLE
Feat(Task-19): As an User, I want to know what I'm about to do

### PR DESCRIPTION
**Pull request by SamuFigueirido**

### CODE REVIEW

**Review by eloydotco — APPROVED** [review]

File: src/app/components/contact-button/contact-dialog.component.html [hunk 1]
```diff
@@ -6,24 +6,43 @@
 <form class="form-dialog" autocomplete="off" layout-align="center center">
     <md-input-container class="user-container" layout="row" layout-align="left">
         <label>Name</label>
+        <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.html]: Bottom is the default configuration for md-direction so you dont need `md-direction="bottom"` in this case

File: src/app/components/contact-button/contact-dialog.component.html [hunk 2]
```diff
@@ -6,24 +6,43 @@
...
+            Your name
+        </md-tooltip>
         <md-icon class="material-icons">account_circle</md-icon>
         <input ng-model="name" type="text" required autocomplete="off" />
     </md-input-container>
 
     <md-input-container class="subject-container" layout="row" layout-align="left">
         <label>Subject</label>
+        <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.html]: Same

File: src/app/components/contact-button/contact-dialog.component.html [hunk 3]
```diff
@@ -6,24 +6,43 @@
...
+            The subject of the issue
+        </md-tooltip>
         <md-icon class="material-icons">short_text</md-icon>
         <input ng-model="subject" type="text" required autocomplete="off" />
     </md-input-container>
 
     <md-input-container class="content-container" layout-align="left">
         <label>Issue</label>
+        <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.html]: Same

File: src/app/components/contact-button/contact-dialog.component.html [hunk 4]
```diff
@@ -6,24 +6,43 @@
...
+        </md-tooltip>
         <md-icon class="material-icons">comment</md-icon>
         <textarea ng-model="issue" type="text" required autocomplete="off" max-rows="3" rows="3"></textarea>
     </md-input-container>
 
     <div class="buttons-container" layout="row" layout-align="center">
-        <md-button class="md-primary btn-submit" type="submit" ng-click="$ctrl.submit()">Submit</md-button>
-        <md-button class="md-warn btn-cancel" type="reset" ng-click="$ctrl.cancel()">Cancel</md-button>
+        <md-button class="md-primary btn-submit" type="submit" ng-click="$ctrl.submit()">
+            <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.html]: Same

File: src/app/components/contact-button/contact-dialog.component.html [hunk 5]
```diff
@@ -6,24 +6,43 @@
...
+                Submit
+            </md-tooltip>
+            Submit
+        </md-button>
+        <md-button class="md-warn btn-cancel" type="reset" ng-click="$ctrl.cancel()">
+            <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/contact-button/contact-dialog.component.html]: Same

File: src/app/components/container-days/container-days.component.html [hunk 6]
```diff
@@ -4,20 +4,47 @@
...
+
+                                <a class="temp">
+                                    <md-tooltip md-direction="right" md-delay="400">
+                                        Temperature in degrees grades
+                                    </md-tooltip>
+                                    {{dayHour.temperature}}°C
+                                </a>
                             </div>
                             <div class="weather-info">
+                                    <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/container-days/container-days.component.html]: Same

File: src/app/components/list-cities/list-cities.component.html [hunk 7]
```diff
@@ -1,7 +1,12 @@
 <div class="container-fluid" layout="row" layout-align="center center">
     <ul class="list-cities">
         <li class="list-cities" ng-repeat="citySearched in $ctrl.listCities track by $index">
-            <a ui-sref="mainPage({city: citySearched})">{{citySearched}}</a>
+            <a ui-sref="mainPage({city: citySearched})">
+                <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/list-cities/list-cities.component.html]: Same

File: src/app/components/logo-letters/logo-letters.component.html [hunk 8]
```diff
@@ -1,5 +1,8 @@
-<div class="title-container" layout-align="center center" >
+<div class="title-container" layout-align="center center">
     <div class="title" ui-sref="startPage" ui-sref-active="active">
+        <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/logo-letters/logo-letters.component.html]: Same

File: src/app/components/search-input-button/search-input-button.component.html [hunk 9]
```diff
@@ -1,9 +1,15 @@
 <form autocomplete="off" layout="row" layout-align="center center" ng-submit="$ctrl.searchCity($ctrl.city)">
     <md-input-container class="input-container">
-        <label>{{isSpanish ? 'Mostrarme el tiempo en...' : 'Show me the weather in...'}}</label>
+        <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/search-input-button/search-input-button.component.html]: Same

File: src/app/components/search-input-button/search-input-button.component.html [hunk 10]
```diff
@@ -1,9 +1,15 @@
...
+            Input to introduce a city
+        </md-tooltip>
+        <label>Show me the weather in...</label>
         <input type="text" ng-model="$ctrl.city" />
     </md-input-container>
     <md-button ng-click="$ctrl.searchCity($ctrl.city)" class="md-fab md-mini md-raised:not">
+        <md-tooltip md-direction="bottom" md-delay="400">
```
Comment (gbtec-ext-jorgevillar) [inline src/app/components/search-input-button/search-input-button.component.html]: Same

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Pull request merged**